tests/js/client/server_parameters/test-telemetrics-true-auth.js

```
@@ -0,0 +1,777 @@
+
+const jwtSecret = 'abc';
+
+if (getOptions === true) {
+  return {
+    'server.enable-telemetrics-api': 'true',
+    'server.authentication': 'true',
+    'server.jwt-secret': jwtSecret,
+  };
+}
+
+let jsunity = require('jsunity');
+let internal = require('internal');
+const FoxxManager = require('@arangodb/foxx/manager');
+const path = require('path');
+const basePath = path.resolve(internal.pathForTesting('common'), 'test-data', 'apps', 'redirect');
+let arangodb = require('@arangodb');
+const users = require('@arangodb/users');
+const isCluster = require("internal").isCluster();
+const isEnterprise = require("internal").isEnterprise();
+let smartGraph = null;
+if (isEnterprise) {
+  smartGraph = require("@arangodb/smart-graph");
+}
+const userName = "abc";
+const databaseName = "databaseTest";
+const cn = "testCollection";
+const cn2 = "smartGraphTestCollection";
+const cn3 = "edgeTestCollection";
+const vn1 = "testVertex1";
+const vn2 = "testVertex2";
+
+let db = arangodb.db;
+
+function createUser() {
+  users.save(userName, '123', true);
+  users.grantDatabase(userName, "_system", 'ro');
+  users.reload();
+}
+
+function removeUser() {
+  users.remove(userName);
+}
+
+function getTelemetricsResult() {
+  try {
+    let res;
+    let numSecs = 0.5;
+    while (true) {
+      res = arango.getTelemetricsInfo();
+      if (res !== undefined || numSecs >= 16) {
+        break;
+      }
+      internal.sleep(numSecs);
+      numSecs *= 2;
+    }
+    assertNotUndefined(res);
+    return res;
+  } catch (err) {
+  }
+}
+
+function getTelemetricsSentToEndpoint() {
+  try {
+    let res;
+    let numSecs = 0.5;
+    while (true) {
+      res = arango.sendTelemetricsToEndpoint("/test-redirect/redirect");
+      if (res !== undefined || numSecs >= 16) {
+        break;
+      }
+      internal.sleep(numSecs);
+      numSecs *= 2;
+    }
+    assertNotUndefined(res);
+    return res;
+  } catch (err) {
+  }
```

This try ... catch unconditionally silences all errors. It should not be there.